apps/frontend/src/app/components/menu/AuthenticatedMenuItems.tsx

```diff
@@ -17,6 +17,11 @@ export const AuthenticatedMenuItems: React.FC = () => {
           <IonLabel>Dashboard</IonLabel>
         </IonItem>
       </IonMenuToggle>
+      <IonMenuToggle autoHide={false}>
+        <IonItem routerLink={Routes.Dashboard}>
```

Comment:
I believe this routerlink should point at the new artifact page rather than the dashboard

Reply:
Whoops forgot to remove anything associated with the new artifact button, that will take place once dashboard is done per your suggestion on dev workflow